package_control/package_manager.py

```diff
@@ -1754,7 +1754,8 @@ def print_messages(self, package, package_dir, is_upgrade, old_version, new_vers
 
         def read_message(message_path):
             with open_compat(message_path, 'r') as f:
-                return '\n  %s\n' % read_compat(f).rstrip().replace('\n', '\n  ')
+                lines = read_compat(f).rstrip().split('\n')
+                return '\n' + '\n'.join(list(map(lambda s: '  ' + s if s else '', lines)))
```

Comment:
This is a better case for a generator comprehension than a filter call with a lambda. (Also `join` accepts an iterable.)